easybuild/easyconfigs/r/R/R-3.6.0-intel-2019a.eb

```diff
@@ -0,0 +1,2195 @@
+    }),
+    ('roxygen2', '6.1.1', {
+        'checksums': ['ed46b7e062e0dfd8de671c7a5f6d120fb2b720982e918dbeb01e6985694c0273'],
+    }),
+    ('git2r', '0.25.2', {
+        'checksums': ['e002c44551f021a7a1fabce9cbd6ac6f16f6dee4d2c60f4b8c280b9fcddf7796'],
+    }),
+    ('rversions', '2.0.0', {
+        'checksums': ['b50c321d9e973284ae6b1d0c89bd46a40f5174de51fb28e3c77cd12ef34f6f56'],
+    }),
+    ('xopen', '1.0.0', {
+        'checksums': ['e207603844d69c226142be95281ba2f4a056b9d8cbfae7791ba60535637b3bef'],
+    }),
+    ('sessioninfo', '1.1.1', {
+        'checksums': ['166b04678448a7decd50f24afabe5e2ad613e3c55b180ef6e8dd7a870a1dae48'],
+    }),
+    ('rcmdcheck', '1.3.3', {
+        'checksums': ['1ab679eb1976d74cd3be5bcad0af7fcc673dbdfd4406bbce32591c8fddfb93b4'],
+    }),
+    ('remotes', '2.0.4', {
+        'checksums': ['379866c034b95ffb0ce3c77569ec36491d4484c21d5b5af61728b31ae52476c9'],
+    }),
+    ('fs', '1.3.1', {
+        'checksums': ['d6934dca8f835d8173e3fb9fd4d5e2740c8c04348dd2bcc57df1b711facb46bc'],
+    }),
+    ('clisymbols', '1.2.0', {
+        'checksums': ['0649f2ce39541820daee3ed408d765eddf83db5db639b493561f4e5fbf88efe0'],
+    }),
+    ('ini', '0.3.1', {
+        'checksums': ['7b191a54019c8c52d6c2211c14878c95564154ec4865f57007953742868cd813'],
+    }),
+    ('gh', '1.0.1', {
+        'checksums': ['f3c02b16637ae390c3599265852d94b3de3ef585818b260d00e7812595b391d2'],
+    }),
+    ('usethis', '1.5.0', {
+        'checksums': ['4fe91458ebc664193e1593d4623370578114bb356f9cd22956fa1a71869ac65e'],
+    }),
+    ('devtools', '2.0.2', {
+        'checksums': ['99a2fa957068254b8ecdb3fc2d50e2950230910ea31c314fc0e7d934e4bd1709'],
+    }),
+    ('Rook', '1.1-1', {
+        'checksums': ['00f4ecfa4c5c57018acbb749080c07154549a6ecaa8d4130dd9de79427504903'],
+    }),
+    ('Cairo', '1.5-10', {
+        'patches': ['Cairo-1.5-10.patch'],
+        'checksums': [
+            '7837f0c384cd49bb3342cb39a916d7a80b02fffbf123913a58014e597f69b5d5',  # Cairo_1.5-10.tar.gz
+            'da36002db4d70583ef3c772ae56b507f19970d16fd35c412ce52b19ffb6466d8',  # Cairo-1.5-10.patch
+        ],
+    }),
+    ('RMTstat', '0.3', {
+        'checksums': ['81eb4c5434d04cb66c749a434c33ceb1c07d92ba79765d4e9233c13a092ec2da'],
+    }),
+    ('Lmoments', '1.3-1', {
+        'checksums': ['7c9d489a08f93fa5877e2f233ab9732e0d1b2761596b3f6ac91f2295e41a865d'],
+    }),
+    ('distillery', '1.0-6', {
+        'checksums': ['4910e2952f767c1062d7cbe648c90a97009e2b3da316c6b33f6d022cd38b23d6'],
+    }),
+    ('extRemes', '2.0-10', {
+        'checksums': ['2c96ae999b0a64f38e5f39e6e913009f18aa1e7f6cd4054b32a6097c9f38d221'],
+    }),
+    ('pixmap', '0.4-11', {
+        'checksums': ['6fa010749a59cdf56aad9f81271473b7d55697036203f2cd5d81372bcded7412'],
+    }),
+    ('tkrplot', '0.0-24', {
+        'checksums': ['2873630a37d7ae1e09a5803d9a89ca0494edd83526c7b1860d9246543722f311'],
+    }),
+    ('misc3d', '0.8-4', {
+       
```

Comment:
@vanzod missing extensions that were added here recently:

* `medflex` (see #8680)
* `Rserve` and `spls` (see #8758)